monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/PluginTable.tsx

```diff
@@ -98,16 +100,41 @@ const renderSafetyCell = (params) => {
   );
 }
 
+const renderDescriptionCell = (params) => {
+  const ref = React.useRef();
+  const [isOverflowing, setIsOverflowing] = useState(false);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  React.useEffect(() => {
+    if (ref.current) {
+      setIsOverflowing(ref.current.scrollWidth > params.api.rootElementRef.current.scrollWidth);
+    }
+  }, [ref]);
+
+  return (
+    <>
+      <Typography noWrap={!isExpanded} ref={ref}>
```

Comment:
I believe Typography is not needed. Just use regular div

Reply:
Div doesn't have the ability to add ellipsis